.gitignore

```diff
@@ -1,18 +1 @@
-# Build and Release Folders
-bin-debug/
-bin-release/
-[Oo]bj/
-[Bb]in/
-
-# Other files and folders
-.settings/
-
-# Executables
-*.swf
-*.air
-*.ipa
-*.apk
-
-# Project files, i.e. `.project`, `.actionScriptProperties` and `.flexProperties`
-# should NOT be excluded as they contain compiler settings and other important
-# information for Eclipse / Flash Builder.
+node_modules

```

Comment:
Your `gitignore` should be like:
 ```
node_modules
test.md
```
Because `test.md is a test to prove the power of using the `.gitignore` file according to the project requirements that Microverse gave you.

- [ ] Add `test.md` in your `.gitignore` file.